controllers/os-suse-jeos/pkg/generator/script_tests.go

```diff
@@ -15,19 +15,23 @@
 package generator
 
 import (
+	"os"
```

Comment:
Rename this file to `controllers/os-suse-jeos/pkg/generator/script_test.go` removing the plural `s` at the end.
Otherwise it is execute with the productive code end `os.Setenv` overwrites the `BOOT_COMMAND` and  `OS_CONFIG_FORMAT` envs all the time.